src/main/scala/no/ndla/imageapi/ImageApiProperties.scala

```diff
@@ -91,18 +91,25 @@ object ImageApiProperties extends LazyLogging {
   private def getBoolean(envKey: String): Boolean = {
     get(envKey).toBoolean
   }
-
 }
 
-object PropertiesLoader {
+object PropertiesLoader extends LazyLogging {
   val EnvironmentFile = "/image-api.env"
 
-  def readPropertyFile(): Map[String,Option[String]] = {
-    Source.fromInputStream(getClass.getResourceAsStream(EnvironmentFile)).getLines().map(key => key -> scala.util.Properties.envOrNone(key)).toMap
+  def readPropertyFile() = {
+    Try(Source.fromInputStream(getClass.getResourceAsStream(EnvironmentFile)).getLines().map(key => key -> Properties.envOrNone(key)).toMap)
```

Comment:
Siden du styrer lesing av secrets kun for miljøer som ikke er local nedenfor. Burde det ikke være symmetrisk slik at du kun leser env-fil dersom miljøet er local her?
Liker godt at disse tingene faktisk håndteres her (og ikke at vi må huske å ikke sende med fil i miljøer etc). Men kunne vi klart å lage en util av det? Slik at vi slipper å implementere akkurat den logikken for hvor ting leses fra mange ganger?